docs/downloads/automation-library/integrations/LinearBAI/describe-pr-bb.cm

```diff
@@ -0,0 +1,13 @@
+manifest:
+  version: 1.0
+
+automations:
+  linearb_ai_description:
+    # skip description for Draft PRs and PRs from bots
+    if:
+      - {{ not pr.draft }}
```

Comment:
we still not support `DRAFT` in BB. This feature released only about 2 weeks ago